test/integration/composite-weighted/composite-weighted.test.ts

```diff
@@ -0,0 +1,55 @@
+import { MeasureReport } from 'fhir/r4';
```

Comment:
This is our first integration test with measure reports instead of detailed results, but I think it's really useful especially for these composites. We should think about a strategy to potentially include some measure report and detailed results testing across the integration tests.

Reply:
Is this a good end of sprint activity (happy to update the list), or do we want to tackle it more formally?

Reply:
Good end of sprint thing I think, just adding/expanding integration tests here in general